tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/locales/scriptToExemplars.txt

```diff
@@ -0,0 +1,26 @@
+# Contains sample aux exemplars for a hint in examples, for use in ScriptToExemplars.java
+# It is generated from the different sets of exemplars that share the same script.
+#   Eg, if locale1 and locale2 use the same script, and locale1 has exemplars [abcd] and locale2 has [cdef], then the set will be [af]
+#   If the set is [], it is omitted here.
+# It is cached here to avoid the cost of doing the derivation at runtime; possible because the CLDR exemplars don't change much in the ST, 
+#   and these are only hints.
+# LikelySubtagsTest.java
+# - Contains a test that verifies that this data correspond to the derivation from the CLDR data.
+# - Also has the details of the derivation from CLDR data, and shows how to refresh this data, if CLDR data changes.
+# - NOTE: We may tweak the derivation over time.
+Arab ;	110 ;	[ؠ-ئةثحخذص-غؽف-كه-ْٰٔٙٛٲٶٹ-ځڃ-ڊڌڍڏڑړڕږژ-ښڠڢڤڦک-ګڭگڱڳڵں-ھہۄۆ-ېےەݜݢݲ{ئا}{ئه}{ئو}{ئى}{ئۆ}{ئۇ}{ئۈ}{ئې}]
+Beng ;	10 ;	[ঌঽৄৎৗৠ-ৣৰ]
+Cans ;	133 ;	[ᐁᐍᐏᐑᐓᐕᐘᐚᐟᐠᐢ-ᐤᐦᐨᐩᐯᐻᐽᐿᑁᑃᑅᑇᑉᑊᑌᑘᑚᑜᑞᑠᑢᑤᑦᑫᑵᑷᑹᑻᑽᑿᒁᒃᒉᒓᒕᒗᒙᒛᒝᒟᒡᒣᒭᒯᒱᒳᒵᒷᒹᒻᒼᓀᓊᓌᓎᓐᓕ-ᓘᓚᓛᓪᓭᓷᓹᓻᓽᓿᔁᔃᔅᔦᔰᔲᔴᔶᔸᔺᔼᔾᕆ-ᕉᕋᕌᕐᕕ-ᕚᕝᕿ-ᖃᖅᖏᖑ-ᖖᖠ-ᖦᙱ-ᙶ]
+Cyrl ;	111 ;	[ʼве-йфцш-яё-ќўџѡѣѧѫѯѱѳѵѷѻѽѿ҃҇ґғҕҗҙқҝҟҡңҥҩҫҭүұҳҵҷҹһҽҿӑӕӗәӡӣөӯӳӷԥⷠ-ⷪⷬⷭⷯⷱⷴⸯꙁꙋꙍꙗ꙽ꙿ{гӏ}{дь}{кӏ}{нь}{пӏ}{тӏ}{хӏ}{цӏ}{чӏ}]
+Deva ;	63 ;	[ँ-ःअ-ऍए-ऑओऔखघङछझञठढणथधफभऱळशष़ऽिू-ॅैॉौ्ॐ-॒ड़ॠ-ॣॻॼॾॿ‌‍{क्ष}{ज्ञ}{त्र}{श्र}]
+Ethi ;	241 ;	[ሇሏሗሟ-ሧሯሷ-ሿቇቐ-ቖቘቚ-ቝቧ-ቯቷ-ቿኇኗ-ኟኧኯኸ-ኾዀዂ-ዅዏዟ-ዧዯዷ-ጇጏጘ-ጟጧ-ጷጿ-ፇፏፗ-ፚ፟ᎀ-ᎏⶀ-ⶖⶠ-ⶦⶨ-ⶮⶰ-ⶶⶸ-ⶾⷀ-ⷆⷈ-ⷎⷐ-ⷖⷘ-ⷞ]
+Grek ;	63 ;	[ἀ-ἇἐ-ἕἠ-ἧἰ-ἷὂ-ὄὐ-ὗὢ-ὧὰὲὴὶὸὺὼᾶῆῒῖῗῢῦῧῶ]
+Hans ;	292 ;	[丧乍乾仑仿伏佉佐佑佤侬俩俱倦偌傣储催兆兑净凉凶刑删券剂剌劣勋勾匆匙卜卧厍厘叉吕吞吠吴吻呈呐呦咤咨咯哟唔唤啰喏喻嗒圜圳坏坑埋塑墙壤壳夥奎妆妖妳姊娜娟嫁孕孜宅宠宫宴寇寡寮寸届岗岘岩岭岳崩帛庄庇庙庞弊彊彦彷後循徵怜恒悄患悦惊愧愿慎扑扥扮扰抑抛拏拘拦拷挂捉掩摇摊摔撑敍敲斜斥於旨晃暮曹栋框梨横樱欠歪毁氛汇泄泛泣泪泾洁津浑浙浴涂涅淋淘添渣渴溜滚澎灾灿烟煌煮燕牠-牢猫珀珥琉琊琳琼璃畅疲疾盈盒盗盲磅祛禄秩稀窭箫粒粹糖纠纬绵缚缠缪缶罐羯翅翠耻聘肖肤胁胶脚腐腔腹腾臂芒芭苗茂茅茨荼莪萍著蒲蓉蓬蔑蔕藉虚虹虾蚁蝎蠢袭触谚谜谨贩贸贼赔赶趁跎踩踪迹递遥遵邓郁酋鉴钵钻铺链锐锺镑镭阇阗阮雯霖韵頞颜飙馨骂骤魁鸭鹤]
+Hant ;	291 ;	[俏倉偽傅傘凋凍刨刮剃勾匯卹唇唵啤喪喲噁噓嚏坑墅墓墟墳壩壺孕孵寺尿屍屑帚幟廈廟忡憊懨扮捏捧掠掰揹搏摀撕撲攀攤敞斑斜斧暮曇曬曳朔杖枯栓栽棍棕棺椒槌橄橇橘橙檬檸櫚櫻欖欠残殭汁沫沮泣浣浴涎涮淇淋湘溜漿澡濕烘烹焊焙焰煮燙爍犀犬狡狸猩猾猿獺獾瓢甕疲疾瘦瘧皂皿盆盈盔盥睏瞇瞌瞪磚礫祈禱禿稻穀窄竿筷箏箔篷籠糖糥紉紮紳綽綿縫繃繡繩纖罈罐罩聾肌肺脈脖腐腹膚膠臂臟芙芭芽苣茄茵茸莓莖菇菌萎萵葵蒜蒸蓄蓉蔔蔥蔬蕉蕾薑薯蘋蘑蘿蚊蚓蚯蜥蜴蝙蝟蝠蝦螂螃螞螺蟀蟄蟋蟑蟳蟻蠅蠕蠣衫袍裏裱裹褐襪襯訝診謎豎豔豹贛跆跪踩軸轎辜辣鄙酪醬鉅鉤鋁錨錶鍊鏈鏢鐡鑿閩阱雌霜静靴鞭頌頸顛颱飆飪餃餌餚餾駝駱驕鬍魷鮑鯊鯨鱷鳩鴨鵡鶴鸚鹽黛鼬龐]
+Hebr ;	29 ;	[ְ-ֹֻ-ֽֿׁׂׄפ{אַ}{אָ}{בֿ}{וּ}{יִ}{כּ}{פּ}{פֿ}{שׂ}{תּ}{ײַ}]
+Latn ;	375 ;	[a-mo-z·ß-öø-ÿāăąćĉċčďđēĕėęěĝğġģĥħĩīį-ıĵķĺļľłńņňŋōŏőœŕŗřśŝşšťŧũūŭůűųŵŷźżžƒƙơưƴǀ-ǃǎǐǒǔǝǥǧǩǫǯǹșțȟɓɔɖɗəɛɣɨɩɲʉ-ʋʒʔʻʼˀḍḑḓḥḵḷḻḽḿṅṇṉṋṛṣṭṯṱṽẁẃẅẓạảấầẩẫậắằẳẵặẹẻẽếềểễệỉịọỏốồổỗộớờởỡợụủứừửữựỳỵỷỹꞌ{a\:}{aʼ}{aː}{a̧}{a̱}{a᷆}{a᷇}{bʼ}{ch}{chʼ}{ch’}{cʼh}{dź}{e\:}{eʼ}{eː}{e̱}{e᷆}{e᷇}{gh}{gy}{gʷ}{gʻ}{g̃}{g̱}{i\:}{iʼ}{iː}{i̧}{i̱}{i᷆}{i᷇}{jhy}{kh}{kʷ}{kʼ}{k’}{ly}{mb}{m̀}{m̄}{nd}{ng}{ny}{n̄}{o\:}{oʻ}{oː}{o̱}{o᷆}{o᷇}{pʼ}{qʼ}{sh}{ts’}{ty}{tz}{tzʼ}{tł’}{tʼ}{t’}{uʼ}{uː}{u̧}{u᷆}{u᷇}{y̱}{à\:}{àː}{á\:}{áː}{è\:}{èː}{é\:}{éː}{ì\:}{ìː}{í\:}{íj́}{íː}{ò\:}{òː}{ó\:}{óː}{ùː}{úː}{ą́}{ę́}{į́}{ŋg}{ŋ̀}{ŋ́}{ŋ̄}{ǝ̀}{ǝ́}{ǝ̂}{ǝ̄}{ǝ̌}{ǫ́}{ɔ̀}{ɔ́}{ɔ̂}{ɔ̃}{ɔ̄}{ɔ̈}{ɔ̌}{ɔ̧}{ɔ̱}{ɔ᷆}{ɔ᷇}{ə̀}{ə́}{ə̂}{ə̄}{ə̌}{ɛ̀}{ɛ́}{ɛ̂}{ɛ̃}{ɛ̄}{ɛ̈}{ɛ̌}{ɛ̧}{ɛ̱}{ɛ̱̈}{ɛ᷆}{ɛ᷇}{ɨ̀}{ɨ́}{ɨ̂}{ɨ̄}{ɨ̌}{ɩ̀}{ɩ́}{ɩ̂}{ʉ̀}{ʉ́}{ʉ̂}{ʉ̄}{ʉ̈}{ʉ̌}{ʊ̀}{ʊ́}{ʊ̂}{ẹ̀}{ẹ́}{ọ̀}{ọ́}]
```

Comment:
Why is plain 'n' missing?

Reply:
We can also tune this in additional PRs (probably for 45). For example, I think the Latin set can be reduced further.